src/adam_core/propagator/propagator.py

```diff
@@ -6,12 +6,16 @@
 import numpy.typing as npt
 import quivr as qv
 
-from adam_core.ray_cluster import initialize_use_ray
-
+from ..coordinates.cartesian import CartesianCoordinates
+from ..coordinates.origin import Origin, OriginCodes
+from ..coordinates.spherical import SphericalCoordinates
+from ..coordinates.transform import transform_coordinates
+from ..dynamics.aberrations import C
```

Comment:
```suggestion
from ..constants import Constants as c

C = c.C
```

Might be a bit cleaner to import the constant from where it's defined.

Reply:
My suggestion will fail linting.